openmdao/components/interp_util/interp_bsplines.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, grid, values, interp=None, **kwargs):
         **kwargs : dict
             Interpolator-specific options to pass onward.
         """
-        super(InterpBSplines, self).__init__(grid, values, interp)
+        super(InterpBSplines, self).__init__(grid, values, interp, **kwargs)
 
         self._vectorized = True
         self.k = self.options['order'] + 1
```

Comment:
Can you add a simple test that fails because the order was not passed correctly?

Reply:
Just pushed one now, thanks!